pages/agent/v3/elastic_ci_aws/managing_elastic_ci_stack.md

```diff
@@ -87,14 +87,15 @@ or a specific release available from the [releases page](https://github.com/buil
 The latest stable release can be deployed to any of our supported AWS Regions.
 
 The most recent build of the CloudFormation stack is published to
-`https://s3.amazonaws.com/buildkite-aws-stack/master/aws-stack.yml`, along with
+`https://s3.amazonaws.com/buildkite-aws-stack/main/aws-stack.yml`, along with
 a version for each commit at
+`https://s3.amazonaws.com/buildkite-aws-stack/main/${COMMIT}.aws-stack.yml`.
+For commits that are ancestors of , the is build was published to
```

Comment:
There's something missing here. Ancestors of what?

Reply:
Fixed @mbelton-buildkite, though I wonder if this should remain in plain text or some special info box is needed. TBF, I don't think we need it to emphasise it that much, and as we release more versions of the stack, this becomes less important.